### DISCUSSION TITLE
Free license for commercial use

### DISCUSSION THREAD

**Discussion by IvanDrag0** [retypeapp/retype#340] [Q&A] [answered]
Does the free license allow for commercial use (even if it's internal only)?

**Comment by geoffreymcgill** [comment, marked as answer]
Hi @IvanDrag0. Correct. Commercial use is permitted. The primary features of Retype Pro are the 1000 page limit and removal of the `Powered by Retype` branding. 

Hope this helps.

**Reply by geoffreymcgill** [reply]
The new `visibility: private` and `visibility: protected` for password protected pages will be included in Retype v3.0, but will be Retype Pro only features.
